apps/explorer/src/ui/Placeholder.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import styles from './utils/customStyles.module.css';
+
+export function Placeholder({
+    width,
+    height,
+}: {
```

Comment:
Props should be pulled into an interface and exported separately like the other base UI components.